lib/rex/proto/apache_j_p.rb

```diff
@@ -0,0 +1,127 @@
+# -*- coding: binary -*-
+
+require 'bindata'
+
+# @see: https://tomcat.apache.org/connectors-doc/ajp/ajpv13a.html
```

Comment:
Not a blocker; Potentially putting a waybackmachine link here would help future travellers 👀

Reply:
https://web.archive.org/web/20231106223828/https://tomcat.apache.org/connectors-doc/ajp/ajpv13a.html